### PR TITLE
Group access manager

### PR DESCRIPTION
**Pull request by nixmen**
Adding class for checking current user entry in group by code

### CODE REVIEW

File: src/GroupAccessManager.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+     */
+    private function __clone()
+    {
+    }
+
+    /**
+     * Get access manager instance
+     * @return static
+     */
+    public static function instance()
```
Comment (senor-artemisio) [inline src/GroupAccessManager.php]: @nixmen should change method name to "getInstance" aka bitrix style look Bitrix\Main\Application class

Comment (nixmen) [inline src/GroupAccessManager.php]: Fixed

File: src/GroupAccessManager.php [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+use Bex\Tools\Group\GroupTools;
+
+/**
+ * Class of access manager group
+ * Implements method of checking current user entries in group by code
+ *
+ * Example:
+ *  <?php
+ *      use Bex\Niceaccess\GroupAccessManager;
+ *      $result = GroupAccessManager::instance()->check('group_code');
```
Comment (senor-artemisio) [inline src/GroupAccessManager.php:19]: why instance? must be getInstance

File: src/GroupAccessManager.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+namespace Bex\Niceaccess;
+
+use Bex\Tools\Group\GroupTools;
+
+/**
+ * Class of access manager group
+ * Implements method of checking current user entries in group by code
+ *
+ * Example:
+ *  <?php
```
Comment (senor-artemisio) [inline src/GroupAccessManager.php:17]: bad idea use php code without ``` wrapping. Example 

``` php
---php code here---
```

**Pull request merged**